entities/src/main/java/com/flipkart/varadhi/entities/cluster/ShardStatus.java

```diff
@@ -21,4 +21,15 @@ public boolean canStart() {
         return state == ShardState.STOPPED || state == ShardState.ERRORED || state == ShardState.UNKNOWN ||
                 state == ShardState.STOPPING;
     }
+
+    @JsonIgnore
+    public boolean isStarted() {
+        return state == ShardState.STARTED ||  state == ShardState.STARTING;
```

Comment:
state should be final and non-null checked, else it can lead to npe

Reply:
done